icu4c/source/data/misc/zoneinfo64.txt

```diff
@@ -3440,7 +3325,7 @@ zoneinfo64:table(nofallback) {
   "CV", //Z#341 Atlantic/Cape_Verde
   "FO", //Z#342 Atlantic/Faeroe
   "FO", //Z#343 Atlantic/Faroe
-  "NO", //Z#344 Atlantic/Jan_Mayen
+  "DE", //Z#344 Atlantic/Jan_Mayen
```

Comment:
Hi @yumaoka we got a question about this line here. [I found a statement](http://www.timezoneconverter.com/cgi-bin/zoneinfo.tzc?s=default&tz=Atlantic/Jan_Mayen) that “The Atlantic/Jan_Mayen time zone has been deemed obsolete. It has been replaced by Europe/Oslo.” -- but I wonder why it is now associated with Germany instead of Norway. (@gnrunge FYI)

Reply:
@yumaoka could you please take a look at this? -->

> Hi @yumaoka we got a question about this line here. [I found a statement](http://www.timezoneconverter.com/cgi-bin/zoneinfo.tzc?s=default&tz=Atlantic/Jan_Mayen) that “The Atlantic/Jan_Mayen time zone has been deemed obsolete. It has been replaced by Europe/Oslo.” -- but I wonder why it is now associated with Germany instead of Norway. (@gnrunge FYI)

Reply:
Thanks @yumaoka for fixing this in PR #2186!